.licenserc.yaml

```diff
@@ -79,6 +78,11 @@ header:
     - 'include/dmlc' # symlink to 3rdparty/dmlc-core/include/dmlc
     - 'include/mshadow' # symlink to 3rdparty/mshadow/mshadow
     - 'include/mkldnn' # symlinks to 3rdparty/mkldnn
+    - 'include/nnvm' # symlinks to 3rdparty/tvm/nnvm/include/nnvm
+    - 'example/automatic-mixed-precision/common' # symlinks to example/image-classification/common
+    - 'example/quantization/common' # symlinks to example/image-classification/common
+    - 'scala-package/packageTest/core/scripts' # symlinks to scala-package/core/scripts
+    - 'scala-package/packageTest/examples/scripts' # symlinks to scala-package/examples/scripts
```

Comment:
do we know why this is being added?

Reply:
The license check utility [skywalking eyes](https://github.com/apache/skywalking-eyes) has been updated to support detecting more license issues and recently started failing on these symlinks. Thus, adding them here. :)

Reply:
skywalking eyes is such a cool name!